.github/workflows/ci.yml

```diff
@@ -353,3 +353,16 @@ jobs:
         env:
           TWINE_USERNAME: __token__
           TWINE_PASSWORD: ${{ secrets.pypi_token }}
+
+      - name: get wasm dist artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: wasm_wheels
+          path: wasm
+
+      - name: upload to github release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: |
+            wasm/*.whl
+          prerelease: ${{ contains(github.ref, 'alpha') || contains(github.ref, 'beta') }}
```

Comment:
This seems to creates a new release, we want to add the file to the existing (just created) release.

Looks like [this action](https://github.com/marketplace/actions/upload-files-to-a-github-release) should do what we want.

Reply:
That action also create new release if there is no pre-existing one: https://github.com/svenstaro/upload-release-action/blob/fb1eb39e74209484069862c0ad703e495f307650/src/main.ts#L27-L39

I think they behave pretty much the same, but svenstaro/upload-release-action doesn't handle network errors quite well which leads to upload failure sometimes.

Reply:
humm are you sure `softprops/action-gh-release` will add the files to an existing release if it's triggered from that release?

Reply:
I think so.

https://github.com/softprops/action-gh-release/blob/50195ba7f6f93d1ac97ba8332a178e008ad176aa/src/github.ts#L201-L289

Reply:
Thanks, I guess we'll find out...